### DISCUSSION TITLE
The opacity argument of the add_points method make points invisible

### DISCUSSION THREAD

**Discussion by AliRezaBeigy** [pyvista/pyvista#4517] [Q&A]
### Describe the bug, what's wrong, and what you expected.

I found a bug!
The opacity argument of the add_points method makethe points invisible. When setting the opacity to a non-zero value, the point cloud does not appear in the plot.

Expected Behavior: The add_points method should render the point cloud with the specified opacity value, allowing the points to be partially transparent.

Actual Result: The point cloud appears completely opaque, regardless of the specified opacity value. The points are not rendered with the desired transparency.

### Steps to reproduce the bug.

```python
import pyvista as pv
import numpy as np

points = np.random.randn(100, 3)

cloud = pv.PolyData(points)

# Create a plotter and add the point cloud with red color and 0.5 opacity
plotter = pv.Plotter()
plotter.add_points(cloud, color='red', opacity=0.5)

plotter.show()
```

Create a PyVista point cloud object using pv.PolyData.
Use the add_points method to add the point cloud to a plotter.
Set the color argument to a specific color (e.g., 'red').
Set the opacity argument to a non-zero value (e.g., 0.5).
Display the plotter window using show().

### System Information

```shell
--------------------------------------------------------------------------------
  Date: Fri Jun 09 10:39:35 2023 +0330

                OS : Linux
            CPU(s) : 24
           Machine : x86_64
      Architecture : 64bit
               RAM : 15.5 GiB
       Environment : Python
       File system : ext4
       GPU Details : error
  MathText Support : False

  Python 3.10.10 (main, Mar 21 2023, 18:45:11) [GCC 11.2.0]

           pyvista : 0.39.1
               vtk : 9.2.6
             numpy : 1.23.5
        matplotlib : 3.7.1
            scooby : 0.7.2
             pooch : v1.7.0
             PyQt5 : 5.15.9
           IPython : 8.12.0
        ipywidgets : 8.0.6
             scipy : 1.10.1
              tqdm : 4.65.0
        jupyterlab : 4.0.0
         pythreejs : 2.4.2
      nest_asyncio : 1.5.6

  Intel(R) oneAPI Math Kernel Library Version 2021.4-Product Build 20210904
  for Intel(R) 64 architecture applications
--------------------------------------------------------------------------------
```

**Comment by banesullivan** [comment]
I cannot reproduce this, and I think the `GPU Details : error` in your reporting is very telling here. I suspect there is a GPU driver issue. I recommend making sure drives are up to date and that you have the probeler GL libraries installed
